Cargo.toml

```diff
@@ -39,13 +39,13 @@ members = [
 ]
 
 [workspace.dependencies]
-async-broadcast = "0.6.0"
+async-broadcast = "0.7.0"
 async-lock = "3.3"
 async-std = { version = "1.12.0", features = ["attributes"] }
 async-stream = "0.3.5"
 async-trait = "0.1"
-atomic_enum = "0.2.0"
-base64 = "0.21.7"
+atomic_enum = "0.3.0"
+base64 = "0.22.1"
```

Comment:
We could pin to the minor version in most cases... and have the vendors sort it out.